influence_benchmark/RL/KTO_training.py

```diff
@@ -59,6 +59,15 @@ def format_dataset(example):
     dataset = dataset.shuffle()  # type: ignore
     dataset = dataset.map(format_dataset, batched=False)  # type: ignore
 
+    # check how many positive and negative examples we have
+    num_positives = sum(dataset["label"])
+    num_negatives = len(dataset) - num_positives
+    print(f"Number of positive examples: {num_positives}")
+    print(f"Number of negative examples: {num_negatives}")
+    # d/u should be in the range of 1 to 1.33
+    kto_config.desirable_weight = 1.0
+    kto_config.undesirable_weight = num_positives / num_negatives * kto_config.desirable_weight * 0.95
```

Comment:
What's the 0.95 supposed to capture? Why not just have the desirable_weight be 0.95?

Reply:
So the paper says the ratio should be within ]1,4/3[ not inclusive so i didn't want the ratio to be exactly 1? it's would probably be fine if it was 1 and thats what it was before

Reply:
I see, did you end up sweeping this at all?

Reply:
No, although that is a good idea